website_sale_infinite_scroll/views/templates.xml

```diff
@@ -71,6 +72,9 @@
                             </tr>
                         </tbody>
                     </table>
+                    <a href="#top" class="btn-back-to-top d-none"><i
+                            class="fa fa-arrow-up"
```

Comment:
```suggestion
                            class="fa fa-lg fa-arrow-up"
```

**nitpick(optional, non-blocking):** I personally like it more, but my colleagues think that it looks fine just as it is right now, so you can totally ignore this comment 👍🏼